cmd/server/main.go

```diff
@@ -20,6 +23,7 @@ func main() {
 	graphqlHandler := &relay.Handler{
 		Schema: graphql.MustParseSchema(graph.Schema, r),
 	}
+	graphqlHandler.Schema.ToJSON() // force panic if schema is barfed
```

Comment:
@severedsea FYI `graphql.MustParseSchema` can still let invalid/incomplete schema pass through and fail only upon usage. but doing a `graphqlHandler.Schema.ToJSON()` apparently forces it to fail

Reply:
Oh... Thanks! 👍